src/backlight/strategies/exit.py

```diff
@@ -177,14 +179,13 @@ def exit_at_opposite_signals(
     """
 
     def _exit_at_opposite_signals_condition(
-        df: pd.DataFrame, opposite_signals_dict: dict
-    ) -> pd.Series:
-        current_signal = TernaryDirection(df["pred"][0])
-        opposite_signals = opposite_signals_dict[current_signal]
-        return df["pred"].isin(opposite_signals)
+        df: pd.DataFrame, opposite_signals_dict: dict, indx: pd.Timestamp
```

Comment:
If `index` will not override others, please use `index` instead of `indx`.

Reply:
Ok

Reply:
Done, should I merge?